src/components/post/Post.js

```diff
@@ -140,7 +141,7 @@ class Post extends Component {
                 <a
                   target="_blank"
                   rel="noopener noreferrer"
-                  href={upload.metadata.dataset}
+                  href={upload.metadata.dataset ? upload.metadata.dataset : ""}
```

Comment:
don't think this is needed, can we go with the old version (on the left in red?)

Reply:
fixed